llvm/tools/repo2obj/R2OELFSymbolTable.h

```diff
@@ -154,7 +157,8 @@ template <typename ELFT> class SymbolTable {
   static Elf_Half getTargetShndx(Value const &SV);
 
   Value *insertSymbol(pstore::indirect_string const &Name,
-                      llvm::Optional<SymbolTarget> const &Target);
+                      llvm::Optional<SymbolTarget> const &Target,
+                      bool IsWeak = false);
```

Comment:
What’s the reason for the default value for IsWeak?

Reply:
Removed, thanks.